Doras/src/environments/environment.ts

```diff
@@ -3,6 +3,7 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
+  apiUrl: 'http://localhost:4200',
```

Comment:
this should be centralised to a config file so you dont have to change it in different places!